web-client/src/presenter/computeds/showAppTimeoutModalHelper.test.ts

```diff
@@ -18,6 +19,9 @@ describe('showAppTimeoutModalHelper', () => {
   it('does not show the modal due to no user', () => {
     const result = runCompute(showAppTimeoutModalHelper, {
       state: {
+        idleLogoutState: {
+          state: IDLE_LOGOUT_STATES.COUNTDOWN,
+        },
```

Comment:
This test was actually not working (it wasn't setting the idleLogoutState), nor was the behavior it was testing in some cases.